File: src/main/java/no/nav/service/pensjon/regler/to/TrygdetidResponse.java
 package no.nav.service.pensjon.regler.to;


import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.Trygdetid;

/**
 * Dataoverføringsobjekt, returdata, for tjenesten fastsettTrygdetid.
 *
 * @author Lars Hartvigsen (Decisive), PK-12705
 */
public class TrygdetidResponse extends ServiceResponse{

	
	
	/**
	 * Fastsatt trygdetid.
	 */
	Trygdetid trygdetid;

	/**
	 * Fastsatt trygdetid for AP2016 iht. kapittel 20 og AP2025.
	 */
	Trygdetid trygdetidKapittel20;

    /**
     * Fastsatt trygdetid for annet uføretidspunkt.
     */
    Trygdetid trygdetidAlternativ;

	/**
	 * Pakkseddel for beregningene.
	 */
	private Pakkseddel pakkseddel  = new Pakkseddel(true, true);


}

File: src/main/kotlin/no/nav/pensjon/regler/to/TrygdetidResponse.kt
package no.nav.pensjon.regler.to


import no.nav.pensjon.regler.domain.Trygdetid


class TrygdetidResponse : ServiceResponse() {
    /**
     * Fastsatt trygdetid.
     */
    var trygdetid: Trygdetid? = null

    /**
     * Fastsatt trygdetid for AP2016 iht. kapittel 20 og AP2025.
     */
    var trygdetidKapittel20: Trygdetid? = null

    /**
     * Fastsatt trygdetid for annet uføretidspunkt.
     */
    var trygdetidAlternativ: Trygdetid? = null

}